internal/backend/db/dbgorm/dbgorm.go

```diff
@@ -76,6 +76,14 @@ func (db *gormdb) Connect(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("opendb: %w", err)
 	}
+	sqlDB, _ := client.DB()
```

Comment:
please handle the possible error

Reply:
of there is a client with initialized DB (which we opened 3 lines earlier) and we checked for the error 1 line earlier, then we could get DB from it without checking the error. wdyt?

Reply:
if client.DB can return an error, I think we should check it
if there was no possible error, it would have just return the reference I assume

and it's a crucial part of the system, so 3 more lines of error checking is fine